connections/src/app/quest/quest.module.ts

```diff
@@ -19,10 +15,6 @@ import { ReactiveFormsModule } from '@angular/forms';
     QuestRewardsPageComponent,
     QuestRewardsPanelComponent,
     // QuestLandingComponent,
-    GetBadgeComponent,
```

Comment:
Thank you. I realized. We probably needed the feature for what it should look like, and the feature CRUD for checking basic functionality, so the feature will be easier later